web/modules/custom/block_module/src/Services/UserEmail.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\block_module\Services;
+
+/**
+ * Class to implement a custom service.
+ */
+class UserEmail {
+  /**
+   * Variable to store a message.
+   *
+   * @var string
+   */
+  protected $user;
```

Comment:
What is the use of this variable?

Reply:
Thought of implementing a functionality, later did it using local variable.